ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -195,29 +206,28 @@
 {%         set row_default_encapsulation = router_bgp.address_family_evpn.neighbor_default.encapsulation | arista.avd.default("vxlan") %}
```

Comment:
Indentation, 4 more extra spaces to align with in the above ```if```. Same for below statement.